### PR TITLE
indexer: use `%ziggurat` catch up to get initial blocks

### PR DESCRIPTION
**Pull request by hosted-fornet**
Previously, the indexer started indexing from whatever the current block at indexer start time was.

This PR uses the `%ziggurat` `epoch-catchup` endpoint to get blocks since the beginning of time. That endpoint just gives all the `epochs`, which we recurse through and consume one-by-one, adding each to the index.

### CODE REVIEW

File: app/uqbar-indexer.hoon [hunk 1]
```diff
@@ -470,30 +484,57 @@
...
+            $(slots rest.slot, cards new-cards, state new-state)
+        $(epochs rest.epoch, cards new-cards, state new-state)
       ::
           %indexer-block
-        ?~  blk.update  `state  :: TODO: log block header?
-        =*  epoch-num   epoch-num.update
-        =*  block-num   num.header.update
-        =/  new-slot=slot:zig  [header.update blk.update]
+        %+  consume-slot
+        epoch-num.update  [header.update blk.update]
```
Comment (tacryt-socryp) [inline app/uqbar-indexer.hoon:507]: epoch-num.update should be on prior line

File: app/uqbar-indexer.hoon [hunk 2]
```diff
@@ -600,15 +638,25 @@
...
+  /epochs-catchup
+::
 ++  chain-update-wire
   ^-  wire
   /chain-update
 ::
+++  get-epoch-catchup
+  |=  d=dock
+  ^-  card
+  %+  %~  watch  pass:io
```
Comment (tacryt-socryp) [inline app/uqbar-indexer.hoon:652]: `%~` wideform

**Review by tacryt-socryp — APPROVED** [review]

File: app/uqbar-indexer.hoon (hunk 1 above)
Comment (hosted-fornet) [reply app/uqbar-indexer.hoon]: fixed in 8111d27

File: app/uqbar-indexer.hoon (hunk 2 above)
Comment (hosted-fornet) [reply app/uqbar-indexer.hoon]: fixed in 8111d27

**Pull request merged**